tbx/project_styleguide/templates/patterns/molecules/streamfield/blocks/division_signpost_block.html

```diff
@@ -18,7 +18,12 @@ <h2 class="heading heading--two division-signpost__heading">{% firstof card.head
                 </div>
                 {% srcset_image card.image format-webp fill-{540x280,490x280} sizes="(max-width: 598px) 540px, (min-width: 599px) 490px" alt="" %}
                 {# The title and icon need to be on the same line with no whitespace to prevent the arrow being orphaned on a new line #}
-                <a href="{% pageurl card.page %}" class="button-link button-link--{{ card.card_colour }}">{{ card.link_text }}&nbsp;{% include "patterns/atoms/icons/icon.html" with name="arrow-wide" classname="button-link__arrow" %}<span class="sr-only"> about {% firstof card.heading card.page.title %} division</span></a>
+                <a href="{% pageurl card.page %}"
+                   class="button-link button-link--{{ card.card_colour }}"
+                   {% if card.accessible_link_text %}aria-label="{{ card.accessible_link_text }}"{% endif %}
```

Comment:
@helenb @albinazs What are your thoughts on me adding a new field explicitly for accessible link text?

I tried having a descriptive `link_text` (i.e. "Read more on out charity services") on staging but it doesn't look great, and appending `{{ link_text }} about {{ heading }} division` can get a bit clunky depending on the content of `link_text`.